webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsAdminService.scala

```diff
@@ -0,0 +1,47 @@
+package org.knora.webapi.responders.admin
+import zio.Task
+import zio.ZIO
+import zio.ZLayer
+
+import org.knora.webapi.config.AppConfig
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectGetResponseADM
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectIdentifierADM
+import org.knora.webapi.messages.admin.responder.usersmessages.UserADM
+import org.knora.webapi.responders.ActorDeps
+import org.knora.webapi.store.cache.settings.CacheServiceSettings
+
+trait ProjectsAdminService {
+  def getSingleProjectADMRequest(
+    identifier: ProjectIdentifierADM,
+    requestingUser: UserADM
+  ): Task[ProjectGetResponseADM]
+}
+
+final case class ProjectsAdminServiceLive(responder: ProjectsResponderADM) extends ProjectsAdminService {
+
+  /**
+   * Finds the project by its [[ProjectIdentifierADM]] and returns the information as a [[ProjectGetResponseADM]].
+   * Checks permissions whether the [[UserADM]] requesting the project may see the result.
+   *
+   * @param identifier           a [[ProjectIdentifierADM]] instance
+   * @param requestingUser       the user making the request
+   * @return
+   *     '''success''': information about the project as a [[ProjectGetResponseADM]]
+   *
+   *     '''error''': [[NotFoundException]] when no project for the given IRI can be found
```

Comment:
This kind of documentation (with success and error in triplequotes) is new to me. If this is the way we want to document methods in the future when ZIOs are involved, you should document it somewhere and inform the backend team about it, so everybody does it in the same way.

Reply:
I do not yet have the recipe for the "correct" way of documenting the error type of ZIOs and do not dare to claim that this way of documenting is _the_ way to go forward.